Your Easy

kasfreedom opened the pull request:
Now you can remove it from upwork ;)

kasfreedom committed:
Your Easy

Now you can remove it from upwork ;)

epogrebnyak commented:
Getting error here - NullFormatter() not defined 
```
        #2
        #axarr[i].xaxis.set_major_formatter(NullFormatter())
```

kasfreedom commented:
from matplotlib.ticker import NullFormatter

epogrebnyak commented:
It is still a bit awkward on my local machine, but thanks anyways for the formatiing ideas. 
![image](https://cloud.githubusercontent.com/assets/9265326/24764356/c13a9140-1afc-11e7-8e22-8afdbcffb39d.png)

kasfreedom commented:
Yes its is awkward :) that is what i understood from the essay. If you draw a draft of your requried plot (by hand) I can get exactly what you want.

epogrebnyak commented:
We are targeting a pic like below, but with years at bottow. Someone is working on it, you can wait a bit and maybe review the proposed code. 

BTW, any way I can contact you by e-mail or otherwise?
![image](https://cloud.githubusercontent.com/assets/9265326/24765263/cfdcdf84-1aff-11e7-9925-267520cb0349.png)

kasfreedom commented:
feel free to contact me on my email
[image: Inline image 1]
for i in range(num_sub):
    axarr[i].plot(df[pairs[i]])
    axarr[i].legend(pairs[i], loc='upper left')
    axarr[i].grid(True, ls='dotted')
    if i > 0:
        axarr[i].spines['top'].set_visible(False)
    if i < num_sub-1:
        axarr[i].spines['bottom'].set_visible(False)


f.suptitle('someTitle', fontsize=15)
plt.tight_layout(pad=3, h_pad=-2)
f.savefig('tmp.png')



On Thu, Apr 6, 2017 at 6:35 PM, Evgeny Pogrebnyak <notifications@github.com>
wrote:

> We are targeting a pic like below, but with years at bottow. Some one is
> working on it, you can wait a bit and maybe review the proposed code. BTW,
> any way I can contact you by e-mail or otherwise?
> [image: image]
> <https://cloud.githubusercontent.com/assets/9265326/24765263/cfdcdf84-1aff-11e7-9925-267520cb0349.png>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/epogrebnyak/Business-Conditions-Digest-2017/pull/1#issuecomment-292231379>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIof-07fglRNuztLEMB6WKu8pacT6qyDks5rtRQ9gaJpZM4M1t6t>
> .
>



-- 
Dr.Eng. Kassem Abdallah
Fulda - Germany
Mobile: +49(0)15236279420
Email: kas.freedom@gmail.com
Linkedin: https://it.linkedin.com/in/kassem-abdallah-36713317

kasfreedom commented:
feel free to contact me on my email

![plot](https://cloud.githubusercontent.com/assets/9052155/24774962/4327a27e-1b1b-11e7-9097-594bbcabb148.png)

kasfreedom commented:
feel free to contact me on my email

![plot](https://cloud.githubusercontent.com/assets/9052155/24774962/4327a27e-1b1b-11e7-9097-594bbcabb148.png)

epogrebnyak commented:
I saw you code sent by e-mail, but was it here or somewhere else?

kasfreedom commented:
what do you mean?  кстати я говорю по русски ;)

epogrebnyak commented:
Здорово ) 

Я вот этот текст потерял: 
> feel free to contact me on my email [image: Inline image 1] for i in range(num_sub): 
> axarr[i].plot(df[pairs[i]]) axarr[i].legend(pairs[i], loc='upp…

Вроде бы это было в комментах в пул-реквесту, а сейчас не вижу (

kasfreedom commented:
![plot](https://cloud.githubusercontent.com/assets/9052155/24831615/77a7a890-1c9d-11e7-804f-915d8b78b2b0.png)

kasfreedom commented:
import pandas as pd
import matplotlib.pyplot as plt
from mpl_toolkits.axes_grid.axes_grid import Grid
%matplotlib inline

FILE = "BCD.xls"
df = pd.read_excel(FILE, skip=1, header=1, index_col='DATE')
print(df.head())

doc = """GDP5	PROD5
IND	MANUF
STROI	AGRO
OPT	RETAIL
CARGO	PASS
INV	COMM"""
pairs = [x.split("\t") for x in doc.split("\n")]


f = plt.figure(1,figsize=(6,12))
num_sub = len(pairs)
grid = Grid(f, rect=111,
            axes_pad=.04,# controls the distance inbetween subplots
            nrows_ncols=(num_sub,1), 
            share_x=True)

for i in range(0,num_sub):
    grid[i].plot(df[pairs[i]])
    grid[i].legend(pairs[i], loc='upper left')
    grid[i].grid(linestyle='dotted')
    grid[i].axis['top'].set_visible(False)
    grid[i].axis['bottom'].set_visible(False)

grid[0].axis['top'].set_visible(True)      
grid[-1].axis['bottom'].set_visible(True)

f.suptitle('Title', fontsize=15)
plt.tight_layout(pad=3)


f.savefig('plot.png')

kasfreedom commented:
I have deleted my Fork .. so maybe as results my comment was deleted too.

epogrebnyak commented:
That could be a reason -prbably the comment was on your sid eof the fork. Thanks for reinstatinating it )
What about your e-mail? It was there at the bootom of one of the comments. Maybe you can drop a line at e.pogrebnyak@gmail.com?

kasfreedom commented:
I have sent you an email with a subject 'Private' i meant Привет but the autocorrection added the E at the end.
